lib/opener.ts

```diff
@@ -0,0 +1,13 @@
+///<reference path=".d.ts"/>
```

Comment:
This code is the same in AB & NS CLIs. Consider moving it in the common lib and refer it from there